app/services/search/postgres/podcast_episode.rb

```diff
@@ -15,6 +14,7 @@ class PodcastEpisode
         "podcast_episodes.published_at",
         "podcast_episodes.quote",
         "podcast_episodes.reactions_count",
+        "podcast_episodes.slug",
```

Comment:
We need to select this field as well for when we call `podcast_episode.path` above:

https://github.com/forem/forem/blob/642f110aa8fdb39f241196244f11c39fe6658108/app/models/podcast_episode.rb#L68-L72